L1Trigger/L1CaloTrigger/python/L1NNCaloTauProducer_cfi.py

```diff
@@ -0,0 +1,38 @@
+import FWCore.ParameterSet.Config as cms
+
+L1NNCaloTauProducer = cms.EDProducer("L1NNCaloTauProducer",
```

Comment:
The naming convention typically used now is to start with `l1t`, so something like `l1tNNCaloTau` would be a better label.